integration-tests/scripts/create-github-repo.sh

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env bash
+#
+# Summary:
+#   Creates a new GitHub repository using the GitHub REST API.
+#   Can create repositories in user account or organizations.
+#
+# Parameters:
+#   $1: repo_name         - The name of the repository to create (just the name, not owner/name)
+#                          OR full name like "org/repo" to create in an organization.
+#   $2: description       - Optional description for the repository.
+#   $3: private           - Optional. Set to "true" for private repo, "false" for public (default: false).
```

Comment:
When are we ever going to need to create a private github repo as part of this workflow?

Reply:
remove the private repo feature